### PR TITLE
Network-Score: Add test for EntityMismatch Issue #303

### LINKED ISSUE #303: [C4GT] Network-Score: Add tests for `EntityMismatch`
## Description
Subtask under : https://github.com/cord-network/community/issues/7

New testcase can be tested under `cargo test -p pallet-network-score` after adding it in the code.

## Goals

- [ ] Add tests for `EntityMismatch` for pallet/network-score

## Expected Outcome
- [ ] Test should assert for `EntityMismatch` being returned properly in all the possible calls.
- [ ] If there are more than one function returning this error code, all of them should be validated either as one test case or as multiple test cases.
       
## Acceptance Criteria
NA

## Implementation Details
Look at other test cases, and add a test case for the same.
Ex: In pallets/network-membership check method `test_duplicate_member_request` which validates error code `MembershipAlreadyAcquired`

## Mockups / Wireframes
NA

---- 

## Product Name
CORD

## Organization Name
Dhiway

## Domain
Blockchain

## Tech Skills Needed
Rust

## Mentor(s)
@amarts 

## Complexity
[Low]

## Category
[Test]

## Sub Category
[Beginner friendly]

### PR DESCRIPTION
**Pull request by hardiksharma11**
Fixes #303 
Added the test for EntityMismatch.
@vatsa287  @amarts

### CODE REVIEW

File: pallets/network-score/src/tests.rs [hunk 1]
```diff
@@ -159,3 +159,89 @@ fn check_duplicate_message_id() {
 		);
 	});
 }
+
+#[test]
+fn check_entity_mismatch() {
```
Comment (vatsa287) [inline pallets/network-score/src/tests.rs]: nit: Please change to `revise_rating_with_entry_entity_mismatch_should_fail`

Comment (hardiksharma11) [reply pallets/network-score/src/tests.rs]: Done the required changes

**Review by vatsa287 — APPROVED** [review]
LGTM

**Comment by hardiksharma11** [comment]
@amarts Will you please review it and merge if everything looks good.

**Review by amarts — APPROVED** [review]
LGTM

**Pull request merged**